drools-wb-screens/drools-wb-guided-dtable-editor/drools-wb-guided-dtable-editor-client/src/main/java/org/drools/workbench/screens/guided/dtable/client/widget/analysis/checks/base/Checks.java

```diff
@@ -244,6 +247,14 @@ private void assertChecks( final RuleInspector ruleInspector,
         removedChecks.addAll( allChecks.get( removedRuleInspector ) );
         allChecks.remove( removedRuleInspector );
 
+        for ( final RuleInspector ruleInspector : allChecks.keySet() ) {
```

Comment:
It is. I'm planning to do reuse the maps I'm using for indexing for the checks and inspectors. That will remove this.